csharp/ql/src/Security Features/CWE-937/VulnerablePackage.qhelp

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Using a package with a known vulnerability is a security risk that could leave the
+software vulnerable to attack.
```

Comment:
an attack or attacks?

Reply:
I think you could use any of the above suggestions–my preference would be to leave as it is though.

Reply:
I think you could use any of the above suggestions–my preference would be to leave as it is though.

Reply:
Any of the above would work I think–my preference would be to leave as is.